src/styles/styles.css

```diff
@@ -1,18 +1,13 @@
-@import url("//fonts.googleapis.com/css?family=Open+Sans:400,600");
-
+@import url("https://fonts.googleapis.com/css?family=Overpass:300,400,500,700,900");
 
 :root {
   --brand-font: "Overpass";
   --brand-font-mono: "Overpass-mono";
-  --brand-color-neural-black: #1f1f19;
-  --brand-color-neural-white: #ffffff;
-  --brand-color-neural-gray: #eeede4;
-  --brand-color-neural-blue: #ededef;
-  --brand-color-primary-blue: #145cc6;
-  --brand-color-primary-pink: #ffaeae;
+  --brand-color-primary-blue: #145CC6;
   --brand-color-light-blue: #d0def4;
-  --brand-color-light-pink: #ffeeee;
-  --brand-color-accent-red: #fb3728;
+  --brand-font-color-black: #49475F;
+  --brand-font-color-dark: #3E3D54;
+  --sidebar-bg-color: #EDEDEF;
```

Comment:
I included these as a complete list of the brand-defined colors.  It's the same list used in all of our slide decks now.  Please leave them.  The intent is to then reference them from another variable definition for a particular stylesheet that can then be used there.  Eg, `--highlight-color: var(--brand-color-neutral-gray);`.

Reply:
(And yeah, I didn't bring you up to speed on that before you dug in here.  That's my fault for forgetting about it until now.)